src/features/common/Layout/MyForestContextV2.tsx

```diff
@@ -52,15 +52,35 @@ interface MyForestContextV2Interface {
   contributionsMap: Map<string, MyContributionsMapItem> | undefined;
   registrationGeojson: RegistrationGeojson[];
   donationGeojson: DonationGeojson[];
+  treesPlanted: number;
+  areaRestored: number;
+  areaConserved: number;
+  isTargetModalLoading: boolean;
+  setIsTargetModalLoading: SetState<boolean>;
+  treeTarget: number;
+  restoreTarget: number;
+  setRestoreTarget: SetState<number>;
+  conservTarget: number;
+  setConservTarget: SetState<number>;
+  treeChecked: boolean;
+  setTreeChecked: SetState<boolean>;
+  restoreChecked: boolean;
+  setRestoreChecked: SetState<boolean>;
+  conservChecked: boolean;
+  setConservChecked: SetState<boolean>;
   userInfo: UserInfo | null;
   setUserInfo: SetState<UserInfo | null>;
+  isLoading: boolean;
+  setIsLoading: SetState<boolean>;
```

Comment:
We have 3 API calls here. What does `isLoading` signify?

Reply:
I was using this state variable to manage loader which is actually not needed, I will remove it